app/models/person.rb

```diff
@@ -0,0 +1,16 @@
+class Person < ActiveRecord::Base
+  has_many :tasks
+
+  def self.count_completed(person_id)
+    person = Person.find(person_id)
+    tasks = person.tasks
+    completed_at = tasks.where(completed_at: true)
+    return completed_at.length
+  end
+
+  def self.count_not_completed(person_id)
+    person = Person.find(person_id)
+    task_count = person.tasks.length
+    return task_count - count_completed(person_id)
+  end
```

Comment:
Awesome use of logic in the model! I like to use scopes rather than class methods in the model, but it's up to you. Here's some [information about scopes vs. class methods](http://www.justinweiss.com/articles/should-you-use-scopes-or-class-methods/).